google_api_lib/task.py

```diff
@@ -126,155 +165,156 @@ def update_meta_sheet_feeders(self, puzzle_id):
+                                }
+                            },
+                        ]
+                        + [
+                            {
+                                "userEnteredValue": {
+                                    "stringValue": __get_answer_or_blank(puzzle, i)
                                 },
-                                {"userEnteredValue": {"stringValue": puzzle.name}},
-                                {
-                                    "userEnteredValue": {
-                                        "formulaValue": '=HYPERLINK("%s", "sheet")'
-                                        % puzzle.sheet
-                                    }
+                                "userEnteredFormat": {
+                                    "textFormat": {"fontFamily": "Roboto Mono"}
                                 },
-                            ]
-                            + [
-                                {
-                                    "userEnteredValue": {
-                                        "stringValue": __get_answer_or_blank(puzzle, i)
-                                    },
-                                    "userEnteredFormat": {
-                                        "textFormat": {"fontFamily": "Roboto Mono"}
-                                    },
-                                }
-                                for i in range(max_num_answers)
-                            ]
-                        }
-                        for puzzle in feeders
-                    ],
-                },
+                            }
+                            for i in range(max_num_answers)
+                        ]
+                    }
+                    for puzzle in feeders
+                ],
             },
-            {
-                "autoResizeDimensions": {
-                    "dimensions": {
-                        "sheetId": sheet_id,
-                        "dimension": "COLUMNS",
-                        "startIndex": 1,
-                        "endIndex": 3,
-                    },
+        },
+        {
+            "autoResizeDimensions": {
+                "dimensions": {
+                    "sheetId": sheet_id,
+                    "dimension": "COLUMNS",
+                    "startIndex": 1,
+                    "endIndex": 3,
                 },
             },
-        ]
-    }
-    self.sheets_service().spreadsheets().batchUpdate(
-        spreadsheetId=spreadsheet_id, body=body
+        },
+    ]
+    if protected_range_id is None:
+        requests.append(
+            {
+                "addProtectedRange": {
+                    "protectedRange": {
+                        "range": {
+                            "sheetId": sheet_id,
+                        },
+                        "description": "Autogenerated page",
+                        "warningOnly": True,
+                    }
+                }
```

Comment:
Does this block smallboard from editing the sheet? I guess maybe not because it's warningOnly? I think you can maybe put the requesting user in `editors` and then set this to be False for extra strictness.

Reply:
Extra strictness added!